hs/app/reach/Main.hs

```diff
@@ -319,18 +345,27 @@ swap a b src = T.replace ("${" <> a <> "}") b src
 packs :: Show a => a -> Text
 packs = pack . show
 
-reachImages :: [Text]
-reachImages =
-  [ "reach"
-  , "reach-cli"
-  , "react-runner"
-  , "rpc-server"
-  , "runner"
-  , "devnet-algo"
-  , "devnet-cfx"
-  , "devnet-eth"
+-- Left: 3rd party; Right: Reach
+type Image = Either Text Text
```

Comment:
Why does it matter? Just to know the `reachsh` at the front?

Reply:
That and tagging in `reach update` is handled differently, yup.

What I mean RE: tagging is that `reach update` uses `docker pull reachsh/${img}:${latest|x|x.y|x.y.z}` in the case of our own `reachsh` images, whereas we don't need to do that in the case of e.g. `postgres:11-alpine`.